app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
Did you test your task? Does it work?

Reply:
Terribly sorry. Somehow deleted parentheses after .split (was toying w/code a bit after testing). Fixed the issue.